chef-dk.gemspec

```diff
@@ -53,7 +55,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "chef-provisioning", "~> 1.2"
 
-  gem.add_development_dependency "github_changelog_generator", "1.11.3"
+  gem.add_development_dependency "github_changelog_generator"
```

Comment:
seems like we'd atleast want a `~> 1.11` constraint to prevent spontaneous combustion